.github/workflows/test-indexer-debian.yml

```diff
@@ -0,0 +1,36 @@
+name: Test the preserving of security config files upon upgrade - Wazuh indexer - Debian
+on:
+  pull_request:
+    paths:
+      - 'stack/indexer/deb/debian/*'
+  workflow_dispatch:
+
+jobs:
+  Test-security-config-files-preservation-Debian:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Preinstall latest stable version of Wazuh indexer package
```

Comment:
Preinstall `the` latest stable version of `the` Wazuh indexer package

Use Grammarly in all `echo` calls.

Reply:
Done in https://github.com/wazuh/wazuh-packages/commit/f72e2418b5b5ff9c48c5a25bee1fc380bdac07cf